Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:latest as certs
+RUN apk --update add ca-certificates
+
+FROM busybox
```

Comment:
I tried to use scratch, but because we're shelling out to `sops`, we need a 'real' shell.